functions/constants.js

```diff
@@ -1,4 +1,5 @@
-module.exports = slugs = [
+exports.testChannel = '771821709589479505';
+exports.slugs = [
```

Comment:
Is this just for testing or did y'all decide to move from firebase to the constants holding the slugs?

Reply:
@DarrylBrooks97 This is how it'll actually be. `testChannel` is just a variable so it's easier for us to reference different channels in the future. The slugs may be removed in the future once we decide how to structure Firestore.